src/NetscriptFunctions.ts

```diff
@@ -1757,6 +1764,16 @@ export const ns: InternalAPI<NSFull> = {
     currentNode: Player.bitNodeN,
     ownedAugs: new Map(Player.augmentations.map((aug) => [aug.name, aug.level])),
     ownedSF: new Map(Player.sourceFiles),
+    bitNodeOptions: {
```

Comment:
Unless there's some properties that you're skipping (and if so, those should be commented), you can simplify this to
`{ ...Player.bitNodeOptions, activeSourceFiles: new Map(Player.bitNodeOptions.activeSourceFiles) }`